code/src/main/java/com/google/sps/storage/data-storage/Place.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.storage;
+
+/**
+* Store a place's name and id
+*/
+public class Place {
```

Comment:
(Feel free to do this in a follow up Pr, but you could do it now if you wanted)

We'll want meaningful descriptions of each of these entities in a toString() method (so we can have simple logging + clear error reporting). That can be in a follow up PR.